src/Lime.Client.TestConsole/ValidationRules/JsonEnvelopeValidationRule.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Lime.Client.TestConsole.ViewModels;
+using System;
+using System.Globalization;
+using System.Windows.Controls;
+
+namespace Lime.Client.TestConsole.ValidationRules
+{
+    public class JsonEnvelopeValidationRule : ValidationRule
+    {
+        public override ValidationResult Validate(object value, CultureInfo cultureInfo)
+        {
+            try
+            {
+                var inputJson = value?.ToString();
+
+                if (!string.IsNullOrEmpty(inputJson))
+                {
+                    EnvelopeViewModel.Parse(value.ToString());
+                }
+            }
+            catch (ArgumentException exception)
```

Comment:
We should handle any exception, not only `ArgumentException`, here. So a simple `catch` will be better